source/common/protobuf/utility.cc

```diff
@@ -1043,4 +1043,44 @@ void TimestampUtil::systemClockToTimestamp(const SystemTime system_clock_time,
           .time_since_epoch()
           .count()));
 }
+
+void StructUtil::update(ProtobufWkt::Struct& obj, const ProtobufWkt::Struct& with) {
+  auto& obj_fields = *obj.mutable_fields();
+
+  for (auto const& [key, val] : with.fields()) {
```

Comment:
Again here with the `auto const&`; I think you want `const auto&`

Reply:
Fixed in https://github.com/envoyproxy/envoy/pull/16400/commits/cfad0846aa208507d3e5450807bda6512209028c